docs/changelog.rst

```diff
@@ -36,6 +36,52 @@ We use a one-line description of every pull request.
 ..
 .. - [#XXX] TITLE (AUTHOR)
 
+
+.. _pyirf_0p6p0_release:
+
+`0.6.0 <https://github.com/cta-observatory/pyirf/releases/tag/v0.6.0>`__ (2021-01-10)
+-------------------------------------------------------------------------------------
+
+Summary
++++++++
+
+- Released January 10th, 2022
+- 5 Contributors
+
+Contributors
+++++++++++++
+
+- Maximilian Nöthe
+- Michael Punch
+- Michele Peresano
+- Julian Sitarek
+- Gernot Maier
+
+Description
++++++++++++
+
+This release is mainly a maintenance release with few changes to the functionality.
+The main feature is the update to gammapy 0.19 in the tests and the utility functions
+to get gammapy IRF classes from pyirf output.
+
+In the future, we will more directly interface with and use gammapy, now that
+the science tool decision was taken.
+
+
+merged pull requests
+++++++++++++++++++++
+
+- `#164 <https://github.com/cta-observatory/pyirf/pull/164>`_ Modified gammapy.py, now working for me with gammapy 0.19
```

Comment:
me who? :)

Reply:
I just copied the title from github., but you're right, will change